spec/fixtures/apps/remote-control/main.js

```diff
@@ -1,8 +1,6 @@
 const { app } = require('electron');
 const http = require('http');
 const v8 = require('v8');
-// eslint-disable-next-line camelcase
-const promises_1 = require('timers/promises');
```

Comment:
Unintuitively, I think this may be required. There are consistent test failures that I think trace back to this removal.

cc @clavin who might better understand the usage, because I'd also like to better understand why this is needed with no direct usage.

Reply:
reverted